.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+    extends: [
+        'airbnb-typescript',
+        'airbnb/hooks',
+        'plugin:@typescript-eslint/recommended',
+        'plugin:jest/recommended',
+        'plugin:prettier/recommended'
+    ],
+    plugins: ['react', '@typescript-eslint', 'jest'],
+    env: {
+        browser: true,
+        es6: true,
+        jest: true,
+    },
+    globals: {
+        Atomics: 'readonly',
+        SharedArrayBuffer: 'readonly',
+    },
+    parser: '@typescript-eslint/parser',
+    parserOptions: {
+        ecmaFeatures: {
+            jsx: true,
+        },
+        ecmaVersion: 2018,
+        sourceType: 'module',
+        project: './tsconfig.json',
+    },
+    rules: {
+        'linebreak-style': 'off',
+        'prettier/prettier': [
+            'error',
+            {
+                endOfLine: 'auto',
+            },
+        ],
+    },
+};
```

Comment:
@solo-bog, you can configure your IDE for adding additional line in the end of each file